raiden_contracts/deploy/contract_verifier.py

```diff
@@ -109,7 +109,7 @@ def _store_deployment_info(self, services: bool, deployment_info: DeployedContra
             services=services,
         )
         with deployment_file_path.open(mode="w") as target_file:
-            target_file.write(json.dumps(deployment_info))
+            target_file.write(json.dumps(deployment_info, indent=4))
```

Comment:
I'd usually go with 
```suggestion
            target_file.write(json.dumps(deployment_info, indent=2))
```
to save some space in deeply nested structures. But I guess that is personal preference...